Metro-Records/la-metro-dashboard
# Add maintenance DAGs for logs and db metadata

## Overview

This PR introduces maintenance dags to clear logs and database metadata that are more than 30 days old.

### Checklist


## Testing Instructions

 * Verify that you can run the `airflow_db_clean` and `airflow_log_cleanup` DAGs without error
 * Verify that the output of the DAG runs makes sense

Handles #109
